### DISCUSSION TITLE
Compatiblity with nix-community/comma?

### DISCUSSION THREAD

**Discussion by manveru** [Platonic-Systems/mission-control#25] [Q&A]
The default command for mission-control is `,`. That takes precedence over https://github.com/nix-community/comma usually. But it would be nice if mission-control could detect presence of an existing `,` in `PATH` and delegate to it when the given command is unknown.

**Comment by srid** [comment]
mission-control and nix-community/comma are two totally different projects; I don't think the coupling makes sense; it can also be confusing if user types `, foo` and it (due to a typo on flake.nix) delegates to nix-community/comma when in fact the user expects to run 'foo' script from their project.

Maybe we can change the default script name of mission-control from `,` to something else to avoid conflict with that project.
